presidio-analyzer/presidio_analyzer/__main__.py

```diff
@@ -68,7 +69,15 @@ def serve_command_handler(enable_trace_pii,
     logger.info("GRPC started")
 
     logger.info("Creating RecognizerRegistry")
-    registry = RecognizerRegistry()
+    enable_text_analytics_recognizer = \
```

Comment:
the "Creating RecognizerRegistry" Should go below the `enable_text_analytics_recognizer`